scripts/functions/gemset

```diff
@@ -355,6 +355,7 @@ gemset_initial()
 run_gem_wrappers()
 {
   gem_install gem-wrappers >/dev/null &&
+  ([ "$(printf '%s\n' "3.2.0" "$(gem -v)" | sort -V | head -n1)" == "3.2.0" ] && gem pristine gem-wrappers --only-plugins > /dev/null || exit 0) &&
```

Comment:
```suggestion
  gem_wrappers_pristine &&
```
and add function:
```bash
gem_wrappers_pristine()
{
  if [ "$(printf '%s\n' "3.2.0" "$(gem -v)" | sort -V | head -n1)" == "3.2.0" ]
    gem pristine gem-wrappers --only-plugins > /dev/null || return $?
  else
    true
  fi
}
```

Reply:
@mpapis Oh yeah, much better! 🍻

Reply:
@pedrofurtado let us know when its ready for re-review